docs/FAQ.md

```
@@ -0,0 +1,639 @@
+2. **Exporter Configuration**: Verify your exporter is configured correctly
+
+   ```csharp
+   .AddOtlpExporter(options =>
+   {
+       options.Endpoint = new Uri("http://your-collector:4317");
+       // Check endpoint reachability
+   })
+   ```
+
+3. **Cache Usage**: Metrics are only emitted when cache operations occur
+   ```csharp
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)